examples/compilation-specs/patrol_workflow.yaml

```
@@ -0,0 +1,10 @@
+name: patrol_workflow
+cache_root: gcs://my-bucket/ecoscope/cache/dag-runs
+tasks:
+  get_patrol_observations: {}
+  process_relocations:
+    observations: get_patrol_observations
+  relocations_to_trajectory:
+    relocations: process_relocations
+  draw_ecomap:
+    geodataframe: relocations_to_trajectory
```

```suggestion
  draw_ecomap:
    geodataframe: relocations_to_trajectory
  persist_text:
    text: draw_ecomap
```

Here's what using the task added in #89 would look like here, but #89 gives a fuller example in the tests.